fast-af/fast-checkout-woocommerce
# PINT-451 - Add a support tab to the Fast settings page

# Description

Update the Fast settings page to use templates and add a support tab that includes documentation and a link to submit a support ticket.

# Testing instructions

1. Login to the [staging admin](https://fasttestdev.wpcomstaging.com/wp-admin/).
2. Visit the [Fast settings page](https://fasttestdev.wpcomstaging.com/wp-admin/admin.php?page=fast).
3. Verify that all of the tabs are displaying properly.
4. Visit the [Support tab](https://fasttestdev.wpcomstaging.com/wp-admin/admin.php?page=fast&tab=fast_support).
5. Verify that the support tab is displaying properly.

Note: The documentation in the Support tab has been reviewed by Kern Rikhi from the TCS team.

# Checks

@ilkerulutas @brikr this is ready for review.
